src/main/java/com/minecolonies/rankup/util/Constants.java

```diff
@@ -0,0 +1,59 @@
+package com.minecolonies.rankup.util;
+
+/**
+ * Constants for our plugin.
+ */
+public final class Constants
+{
+
+    private Constants()
+    {
+        //Empty private constructor.
+    }
+
+    public static class PlayerInfo
+    {
+        private PlayerInfo()
+        {
+            //Empty private constructor.
+        }
+
+        public static final String PLAYER_NAME   = "{player}";
+        public static final String PLAYER_GROUP  = "{group}";
+        public static final String PLAYER_RANK   = "{rank}";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1213)